src/components/avatar/Avatar.a11y.spec.jsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import {render, screen} from '@testing-library/react';
+import Avatar from './Avatar';
+import {testA11y} from '../../axe';
+import userEvent from '@testing-library/user-event';
+
+describe('Avatar a11y', () => {
+  it('renders empty avatar ', async () => {
+    const alt = 'alt';
+    const avatar = render(<Avatar alt={alt} />);
+
+    await testA11y(avatar.container);
+    expect(avatar.queryByRole('img')).toBeFalsy();
+    expect(avatar.queryByAltText(alt)).toBeFalsy();
+  });
+
+  it('renders empty avatar with link and label', async () => {
```

Comment:
What happens if link with label will be provided along with alt text for image? Is it ok? Can we test it?

Reply:
Yes, this is a correct usage. Ex. link to another user's profile (label: "User's profile", alt text: "John Doe"). I'll add a test for that case.